plugins/modules/vmware_guest.py

```diff
@@ -575,13 +582,14 @@
             description:
             - Auto logon after virtual machine customization.
             - Specific to Windows customization.
-            default: False
```

Comment:
I would keep `default: ` instead of sentence.

Reply:
I did this at first. But then we'll have to declare the default also in `argument_spec`, otherwise there will be a sanity error. And _this_ means `customization.autologon` will always be set and, therefor, customization will be done.

You see, we can't tell these two cases apart: The user defined customization but not autologon (wants to customize) and the user didn't define customization at all (doesn't want to customize) because `customization.autologon` will always be set.

I've implemented your other suggestions, though.

Reply:
@Akasurde Actually, the original code doesn't even default to `False`:

https://github.com/ansible-collections/community.vmware/blob/77adcf15b42e20c60c0311186d3d37bf0519942d/plugins/modules/vmware_guest.py#L2316-L2318

This looks like `autologon` is ignored if unset. I think I'll just remove any reference to a default from the documentation.